katalog/prometheus-operated/kube-prometheus-rules.yml

```diff
@@ -7,63 +7,68 @@ apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
   labels:
+    app.kubernetes.io/component: exporter
+    app.kubernetes.io/name: kube-prometheus
+    app.kubernetes.io/part-of: kube-prometheus
     prometheus: k8s
     role: alert-rules
   name: kube-prometheus-rules
+  namespace: monitoring
```

Comment:
Avoid adding the `namespace` name. It is injected via the `kustomization.yaml` file.